samples/hello_world/src/main.c

```diff
@@ -5,17 +5,20 @@
  */
 
 #include <zephyr/kernel.h>
+#include <zephyr/drivers/counter.h>
```

Comment:
Reiterating that we should at some point in time revert to the original hello_world main code and have our code in the boards maybe and compile and run from there